script/model/mini_trainer.py

```diff
@@ -57,7 +57,7 @@ def _train_data(self, data):
             transformers.append(modeling_transformer)
 
         error_bias = 1
-        min_percentage_error = 1
+        min_percentage_error = 2
```

Comment:
Just double-checking to make sure this is intended?

Reply:
Yes, it's intended. Though an error higher than 1 is pretty bad...